### PR TITLE
Redesign /firefox/enterprise page (Fixes #7491)

### LINKED ISSUE #7491: new /enterprise page: code & code review
## Description

Code /enterprise, using 100% Protocol components, building those as required to do so

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] Is built fully in Protocol
- [x] Modular layout (for easier future iteration w. content)
- [x] Primary CTA is to download
- [ ] Is live during the week of Sept 16th - 22nd
- [ ] Target is Tier 1 locales but  . . . (Comment from Romain: If en-US can launch first and not let loc hold up the process)

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [x] Code
- [x] Identify analytics requirements and include in page - @sghoseWI 
- [x] Create redirects:  /organizations & /organizations/all to firefox/enterprise.
- [ ] ~Set up hosting for [onboarding guide .pdf,](https://drive.google.com/file/d/1sbF9ghhFa2fZek7NFAjXJabak3R02Enl/view) linked to in /enterprise~ (this is no longer valid - we will not host this guide, which will be included in the download bundles to which we will link as provided by the Enterprise team)
-  [ ] Extract strings to Pontoon, opt-in, none required for launch

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Updates `/firefox/enterprise/` page content.
- Redirects legacy `/firefox/organizations/` page to `/firefox/enterprise/`.
- Updates functional tests for enterprise page.

## Issue / Bugzilla link
#7491

## Testing
Demo: https://www-demo2.allizom.org/en-US/firefox/enterprise/

- [ ] Test responsiveness for LTR / RTL
- [ ] Test comparison list at large / small screen sizes.
- [ ] No regressions to signup/thanks/sla pages.
- [ ] Redirect works as expected.
- [ ] Package download links serve the correct files.

Successful test run: https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/354/pipeline

### CODE REVIEW

**Comment by alexgibson** [comment]
Ok, this is still blocked on download package links, but is close enough to complete for code review.

**Comment by alexgibson** [comment]
Note that this is also still waiting on some GA data-attributes that are to come in https://github.com/mozilla/bedrock/issues/7572

**Comment by alexgibson** [comment]
Analytics are now in place.

**Comment by devinreams** [comment]
@alexgibson sorry I commented on the commit and not the PR itself but after chatting with Tyler a bit, had just a few links and ideas that hopefully are worth incorporating here and now.

I'm happy to come back through later with a separate PR (against this branch or default) if you prefer.

File: bedrock/firefox/templates/firefox/enterprise/index.html [hunk 1]
```diff
@@ -25,74 +23,242 @@
...
+      <ul class="mzp-l-card-third enterprise-download-lists">
+        {% call picto_card(
+          title=_('Windows 64-bit'),
+          class='enteprise-download-package win64',
+          heading_level=3,
+          custom_desc=True
+        ) %}
+          <ul class="mzp-u-list-styled">
+            <li>{{ _('The latest Firefox browser') }}</li>
+            <li>{{ _('The latest Firefox ESR') }}</li>
```
Comment (devinreams) [inline bedrock/firefox/templates/firefox/enterprise/index.html]: Clay [had written this](https://docs.google.com/document/d/1nbSDIpMxmkLcy9Uts54IMB9LVZiTFGpwHRrC-_AFmD8/edit) as "Extended Support Release (ESR)" in the first column (64 bit) so we defined the abbreviation on the page the first time it appeared.

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/enterprise/index.html]: @devinreams thank you for this. Please try and keep feedback consolidated in the gdoc, just so we can keep things centralised.

**Comment by devinreams** [comment]
>Please try and keep feedback consolidated in the gdoc

OK, no problem. Words are in the doc:
https://docs.google.com/document/d/1tNKPHZwxRSRBkhCrDplxDC7-NR3qHSYJnW9A_iFYRfM/edit#

**Comment by alexgibson** [comment]
Download links are now in place. I don't think there are any remaining blockers to shipping this page live. There is still ongoing design discussion happening in the gdoc, but I don't think it's anything large enough to warrant holding up this page any further.

**Comment by alexgibson** [comment]
Copy updates are made. There's now an ask that this page goes out on 9/18, so adding back the DNM label.

**Comment by alexgibson** [comment]
We have the go-ahead to ship this as soon as it's bee reviewed vs waiting 2 weeks.

**Pull request merged**